pkg/set/set.go

```diff
@@ -0,0 +1,29 @@
+package set
```

Comment:
Just a note: there is a [map](https://pkg.go.dev/maps) package in the standard library, that may cover your needs